File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.swift
import Foundation

func solution(_ s:String) -> Bool
{
    var cntP = 0
    var cntY = 0
    for i in s{
        if i == "P" || i == "p" {
            cntP += 1
        } else if i == "Y" || i == "y" {
            cntY += 1
        }
    }
    
    if cntP == cntY { return true }
    else { return false }

}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    
    np = 0
    ny = 0
    
    for i in s:
        if i == 'p' or i == 'P':
            np += 1
        elif i == 'y' or i =='Y':
            ny += 1
    
    if np == ny:
        return True
    else:
        return False